src/wat-writer.cc

```diff
@@ -1024,6 +1024,8 @@ void WatWriter::WriteException(const Exception& except) {
 }
 
 void WatWriter::WriteLimits(const Limits& limits) {
+  if (limits.is_shared)
```

Comment:
Should be `if (limits.is_shared) { WriteOpenSpace("shared"); ... WriteCloseSpace(); }`. Naming is not great, sorry... :-)

See `WriteBeginGlobal` for similar thing for mut flag.

Reply:
Done